.travis.yml

```diff
@@ -17,8 +17,18 @@ addons:
       - mysql-server
       - mysql-client
 
+cache:
+  directories:
+    - $HOME/downloads
+
 install: mvn compile
 
+before_script:
+  - |
+    if [ "$BACKEND" == "hbase" ]; then
+      .travis/install-hbase.sh
```

Comment:
prefer to move .travis to hugegraph-dist/src/assembly/travis

Reply:
done